src/app/learn/get-started/advanced/entry-components/index.html

```diff
@@ -175,7 +170,7 @@ <h1>Modal launch page</h1>
     </li>
     <li>
       <p>
-        In the <stache-code>modal</stache-code> folder, add another new file and name it <stache-code>modal-demo-launcher.component.html</stache-code>. This file serves as the template for the button component that launches the modal form. Add the following code:
+        In the <stache-code>modal</stache-code> folder, create a <stache-code>modal-demo-launcher.component.html</stache-code file to serve as the template for the button component that launches the modal form and add the following code:
```

Comment:
missing the > right before "file to serve"

Reply:
Nice catch. I'm surprised that didn't throw an error when I served. ... Fixed.